cfme/tests/infrastructure/test_providers.py

```diff
@@ -118,16 +119,14 @@ def test_ip_required_validation():
 
 
 @pytest.mark.tier(3)
-@pytest.mark.xfail(message='http://cfme-tests.readthedocs.org/guides/gotchas.html#'
-    'selenium-is-not-clicking-on-the-element-it-says-it-is')
-def test_name_max_character_validation():
+def test_name_max_character_validation(request):
     """Test to validate max character for name field"""
-    prov = VMwareProvider(
-        name=fauxfactory.gen_alphanumeric(256),
-        hostname=fauxfactory.gen_alphanumeric(5),
-        ip_address='10.10.10.12')
-    prov.create()
-    prov.delete(cancel=False)
+    provider = providers.setup_a_provider()
```

Comment:
Change this into a fixture, please, and we are good to go :+1: 
https://github.com/ManageIQ/integration_tests/blob/619b53ba62dcdc40e4f77d2e15987ec090b8af93/cfme/tests/configure/test_visual_infra.py#L45